src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -1,9 +1,10 @@
-<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ page contentType="text/html;charset=UTF-8"%>
 <html>
 <head>
-    <title>My team</title>
+    <title>Taxi-service</title>
 </head>
 <body>
-<h1>Hello, mates!</h1>
+<h1>SUCCESS!</h1>
+<h2>There will be a great design here, but not now :(</h2>
```

Comment:
it would be better to have here links to our pages

Reply:
links was added